### ISSUE TITLE
Is there any way to include canvas or other files in "projects"?

### ISSUE THREAD

**Issue by hubery055** [marcusolsson/obsidian-projects#575]
### What would you like to be added?

I want to include all my files in the projects view. I can't find any way to do it by myself. Does this project have this feature? If not, have you considered adding this feature?

### Why is this needed?

Canvas is a part of notes. There are too many canvases in my notebook, and I need a tool like _projects_ to manage them.

**Comment by trucefilms** [comment]
I agree! It would be fantastic to be able to see my canvas documents in projects.

**Comment by Jnolcox** [comment]
Do they not show up if tagged or in the correct folder?

**Comment by marcusolsson** [comment]
Not sure how this would work. What kind of information would you like to see? Projects uses YAML frontmatter to extract data from notes. Since .canvas and other files don't have frontmatter, it would be tricky I think?

**Comment by trucefilms** [comment]
My personal use case is primarily as an overview of all files in a directory, and while YAML frontmatter wouldn't be an option, I would still find it useful for canvas files to be included in the overview.

**Comment by jebeaudoin** [comment]
I hate it when some provides a suggestion for a wonky workaround in my FRs because they don't understand my use case, so I apologize for doing so here. :)

But if the number of canvases is small, you might create a markdown note for each canvas in the folder. If you are using YAML, you can add the obsidian URL for the canvas as a field like so:

`
link: "[Dashboard](obsidian://open?vault=vault-name&file=canvas-name.canvas)"
`

(Right-clicking on the canvas in the file navigation sidebar will enable you to easily copy the Obsidian URL, I think)

If you then configure the link field in a Project table view to "Enable rich text formatting" the field will be clickable and you can open the canvas directly from the table view.

An imperfect workaround, but perhaps it is useful until a better solution is found?

Cheers,
Jack

**Comment by HHMathewChan** [comment]
> I hate it when some provides a suggestion for a wonky workaround in my FRs because they don't understand my use case, so I apologize for doing so here. :)
> 
> But if the number of canvases is small, you might create a markdown note for each canvas in the folder. If you are using YAML, you can add the obsidian URL for the canvas as a field like so:
> 
> `link: "[canvas-name](obsidian://open?vault=vault-name&file=canvas-name.canvas)"`
> 
> (Right-clicking on the canvas in the file navigation sidebar will enable you to easily copy the Obsidian URL, I think)
> 
> If you then configure the link field in a Project table view to "Enable rich text formatting" the field will be clickable and you can open the canvas directly from the table view.
> 
> An imperfect workaround, but perhaps it is useful until a better solution is found?
> 
> Cheers, Jack
> 
> EDIT: Even simpler... create the markdown note, drag the selected canvas onto the note, copy the resulting embedded link and add it to your YAML like:
> 
> `link: [[Dashboard.canvas]]`

A decent work around suggestion
